extensions/shorten/README.md

```diff
@@ -0,0 +1,8 @@
+# Shorten
+
+Uses netlify's redirect functionality to make a personal URL shortener.
```

Comment:
Could you mention what exactly happens when you run this command here in the readme? Something like the following? 

<img width="613" alt="CleanShot 2023-04-07 at 11 27 04@2x" src="https://user-images.githubusercontent.com/236451/230659239-eb21a9f4-5323-4f78-b389-73ea4d84f521.png">

Reply:
I believe this command must be run in a local Netlify site, is that correct?